### ISSUE TITLE
Any benchmark hyper-parameters and results?

### ISSUE THREAD

**Issue by csyanbin** [siavash-khodadadeh/MetaLearning-TF2.0#3]
HI siavash,

Thanks for the excellent repo and really clean and neat code for Meta learning in TF2.

I ran some of the experiments using this code, but find that the results are not promising as the paper. 

Can you release some benchmark results or hyperparameters?

**Comment by siavash-khodadadeh** [comment]
Hello,

Can you please specify which experiments you ran?
For the Omniglot, this will give the same result as the paper. For Mini-Imagenet as mentioned in the project home page (since we use auto augment), please use the repository which is a fork of original maml and applies auto-augment.

**Comment by csyanbin** [comment]
Thanks for your reply.
For Omni in this repo I got 83.44 with epoch=10 and the default hyper-parameters (seems different from the paper, i.e., M_mb=25, N_u=1), 
this accuracy is similar to the number reported in your NIPS19 paper. 

For Mini-imagenet, I ran the original maml-version with tf1.14 with autoaugment for iters=59999 (cls_5.mbs_4.ubs_1.numstep5.updatelr0.01hidden32maxpoolbatchnorm), and got 37.02 (39.93 in the paper).

Do the above numbers make sense in your experience?

**Comment by siavash-khodadadeh** [comment]
Yes. The hyperparameters are not the same as in the paper since this is the reimplementation of the code with Tensorflow 2.0. We realized that we can even get better results than in the paper. I guess for the Mini-Imagenet, the results make sense and you should notice that these 1000 tasks are randomly sampled each time and so there will be some variance. However, I myself got better than 37.02 when I ran the experiment. We have done meta-learning rate decay manually in our experiments for the paper as well and that also helps. I am closing this issue here and update the hyperparameters for the other repo.
